doc/src/modules/polys/DomainMatrix.rst

```diff
@@ -0,0 +1,30 @@
+.. _polys-domainmatrix:
```

Comment:
Kindly remove this and add the changes you did in this file, and the index file.
Thanks.